Loader/Config/Settings.lua

```diff
@@ -609,10 +613,12 @@ order = {
 	"AntiHumanoidDeletion";
 	"AntiMultiTool";
 	"AntiGod";
+	"ProtectHats";
+	" ";
```

Comment:
Why is there a blank string here?

Reply:
![image](https://user-images.githubusercontent.com/31796727/225019263-81021d29-049f-4188-9df0-9adf781f6aaa.png)

Reply:
👍